virttest/qemu_vm.py

```diff
@@ -1835,6 +1835,12 @@ def sort_key(dev):
         # initialize iothread manager
         devices.initialize_iothread_manager(params, self.cpuinfo)
 
+        # Add object throttle group
+        for group in params.get("throttle_groups", "").split():
```

Comment:
Please use
```suggestion
        for group in params.objects("throttle_groups"):
```
or `params.object_params(group)` will get wrong value. please refer the usage of `object_params`
https://github.com/avocado-framework/avocado-vt/blob/728b1996cb6f8344c1dfa7db32f0ae442ad09b60/virttest/utils_params.py#L70-L71

Reply:
updated